packages/public-types-lib/src/participant.ts

```diff
@@ -124,19 +125,38 @@ export declare interface IParticipantEndpoint {
   value: string;                                          // URL format for urls, ex: https://example.com:8080/fspcallbacks/, or simply 192.168.1.1:3000
 }
 
-export declare interface IParticipantAccount {
+export declare interface IParticipantAccounts {
   id: string;                                             // uuid of the account (from the external accounts and balances system)
   type: "FEE" | "POSITION" | "SETTLEMENT" | "HUB_MULTILATERAL_SETTLEMENT" | "HUB_RECONCILIATION";
   //isActive: boolean                                     //TODO do we need this?
   currencyCode: string;                                   //TODO move
   debitBalance: string | null;                            // output only, we don't store this here
   creditBalance: string | null;                           // output only, we don't store this here
   balance: string | null;                                 // output only, we don't store this here
+  externalBankAccountId: string | null;
+  externalBankAccountName: string | null;
+}
+
+export declare interface IParticipantAccountsChangeRequest{
+	id: string;
+	accountId: string | null;
+	type: "FEE" | "POSITION" | "SETTLEMENT" | "HUB_MULTILATERAL_SETTLEMENT" | "HUB_RECONCILIATION";
+	currencyCode: string;
+	debitBalance: string | null;
+	creditBalance: string | null;
+  balance: string | null;
+	externalBankAccountId: string | null;
+	externalBankAccountName: string | null;    
+	createdBy: string;
+	createdDate: number;
+	approved: boolean;
+	approvedBy: string | null;
+	approvedDate: number | null;
 }
 
 export declare interface IParticipantActivityLogEntry {
   changeType: "CREATE" | "APPROVE" | "ACTIVATE" | "DEACTIVATE"
-      | "ADD_ACCOUNT" | "REMOVE_ACCOUNT"
+      | "CHANGE_ACCOUNT" | "REMOVE_ACCOUNT"
```

Comment:
These should describe the operator action (not just a change), for example: 
CHANGE_ACCOUNT_BANK_DETAILS

Reply:
Should be CHANGE_ACCOUNT_BANK_DETAILS.